scripts/extract/extractor/extract_caffe2.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+import collections
+
+from caffe2.proto import caffe2_pb2
+from .base_extract import BaseExtrctor
+MODEL_TYPE = 'CAFFE2'
```

Comment:
I saw that we use camel case at https://github.com/caicloud/temp-model-registry/pull/7/files#diff-846c79e7297d926aa85515166b8dddabR9

Can we keep the camel case here too?

Reply:
this code is not modified, copy it from mode-registry(Clever 1.6.0), I think if problem appear, we fix it , but we can keep it now. (minimize change script until we complete basic features)
 

Reply:
SGTM